src/logical/permutator.rs

```diff
@@ -88,8 +88,8 @@ impl Permutator {
 
     /// Creates a [`Permutator`] based on one a list of [`VecT`].
     ///
-    /// The sorting of values is done by taking the first [`VecT`] and sort according to these values.
-    /// If two elements are the equal to each other, the next [`VecT`] will be taken to check if the comparison is different.
+    /// The sorting of values is done by taking the first [`VecT`] and sorting them according to these values.
+    /// If two elements are the equal to each other, the next [`VecT`] is taken to check if the comparison is different.
     /// If all [`VecT`] comparisons result in equality, the original order of the two values is preserved.
```

Comment:
This is just a really wordy way of saying that we do a lexicographic order on the `VecT` columns, isn't it?